api/v1alpha1/clusterextension_types.go

```diff
@@ -149,6 +150,8 @@ const (
 
 	ReasonErrorGettingReleaseState = "ErrorGettingReleaseState"
 
+	ReasonManagementFailed = "ManagementFailed"
```

Comment:
Update this reason to reflect that we were unable to check Health status. `Unverifiable`? `Unassessable`?